jinahub/indexers/storage/MongoDBStorage/requirements.txt

```diff
@@ -0,0 +1,5 @@
+pymongo==3.11.4
+jina[hub]==2.0.4
```

Comment:
We don't add Jina to reqs.txt anymore

Reply:
it exist in `postgresqlstorage` as well? why?

Reply:
Then it needs to be removed

Reply:
It's not in PSQL btw